src/platform/plugins/shared/discover/public/application/main/state_management/discover_data_state_container.ts

```diff
@@ -354,7 +364,7 @@ export function getDataStateContainer({
     };
   }
 
-  const fetchQuery = async (resetQuery?: boolean) => {
```

Comment:
It turns out this parameter `resetQuery` is never actually used when calling `fetchQuery`. We call `refetch$.next('reset')` directly in some places instead.